rpc/manager/manager.proto

```diff
@@ -255,10 +255,16 @@ message VersionInfo2 {
   string version = 2;
 }
 
+message License {
+  string license = 1;
+  string host = 2;
```

Comment:
Probably add a doc comment clarifying that `host` is the server that issued the license (i.e. `SYSTEMA_HOST`).

Reply:
Except is that really even true?  It's set to the manager's `SYSTEMA_HOST`, which isn't necessarily how it was set on the laptop when the license was created.

Reply:
So probably have `telepresence license` shove the host in as part of the secret value? And have the manager read that?